src/ColorScale.js

```diff
@@ -482,6 +513,16 @@ export default class ColorScale extends BaseClass {
   bucketAxis(_) {
     return arguments.length ? (this._bucketAxis = _, this) : this._bucketAxis;
   }
+      /**
+      @memberof ColorScale
+      @desc Determines whether or not to display a midpoint centered Axis. Works on linear, log and jenks scales.
```

Comment:
If we allow `scale: "buckets"` to have this as well (see other comment), then this part of this description would change to "Does not apply to quantile scales"